zkit/src/main.rs

```diff
@@ -419,7 +422,20 @@ fn main() {
             &args.public_json,
         ),
         Command::Verify(args) => verify(&args.vk_file, &args.proof_bin, &args.transcript),
-        Command::GenerateVerifier(args) => generate_verifier(&args.vk_file, &args.sol),
+        Command::GenerateVerifier(args) => {
+            match args.protocal.as_str() {
+                "plonk" => {
+                    plonky::api::generate_verifier(&args.vk_file, &args.sol)
+                }
+                "groth16" => {
+                    groth16::api::generate_verifier(&args.vk_file, &args.sol)
+                }
+                _ => {
+                    println!("unknown protocol");
```

Comment:
you can use panic!("unknown protocol") to handle the exception.

Reply:
done